psutil_agent/tests/test_loop.py

```diff
@@ -0,0 +1,65 @@
+import sched
+import time
+
+import mock
+import testtools
```

Comment:
Not really. I probably looked at aws plugin or something and there somebody used testtools. Should I change it?